src/Samples/NewWindow.Core/App.fs

```diff
@@ -21,34 +21,42 @@ type AppMsg =
   | Window2Close
   | Window2Msg of Window2Msg
 
-
 module App =
-  module Window1 =
-    let get m = m.Window1
-    let set v m = { m with Window1 = v }
-    let map = map get set
-  module Window2 =
-    let get m = m.Window2
-    let set v m = { m with Window2 = v }
-    let map = map get set
-    let mapOutMsg = function
+  module App_Window1 =
```

Comment:
I don't see how this module rename helps.

What do you find confusing about the original naming?

Reply:
Oh, you said in your OP:

> ...so their usage in `App.bindings` is not confused with the actual references to `Window2` in `App.bindings`.

Reply:
My problem was encountering `Window1` and `Window2` as actual window labels and as modules.  That overload required keeping context in mind but with so much else going on in NewWindow.Core my "mental buffer" filled up.  It helped me a lot to make sure each label meant only one thing without relying on context.  :-)

Your code looks elegant and compact.  I just struggled when I couldn't be sure one-name-meant-one-object/function.

Reply:
The ability to use the same name in the same scope for different things is an advantage of F#.  [Naming is (impossibly) hard](http://tysonwilliams.coding.blog/2020-07-15_naming_is_impossibly_hard), so being able to avoid some of it by reusing existing names can improve things.

That said, I am symothtic to naming these modules differently.  Instead of repeating the parent module name though, I think `Window1Module` (using [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation)) would be better.

Reply:
Isn't the use of a module of the same name with a record (paralleling the functionality of a class) commonly used in F#, though? I would (humbly) vote against adding "Module" to the ends of module names.

Reply:
I also prefer to reuse the same name, but I am open to using a different name.

Reply:
@TysonMN   I bring ye tidings from the computer stone-age where single-letter variables where normal.  I now overcompensate using verbose labels.  :-)